packages/esm-ward-app/src/ward-workspace/admission-requests-workspace.scss

```diff
@@ -8,3 +8,6 @@
   grid-template-columns: repeat(auto-fill, minmax(280px, 1fr));
   gap: 1rem;
 }
+.content {
+  padding: spacing.$spacing-05;
```

Comment:
I guess we should also do `layout.$spacing-05` here

Reply:
Done!